cypress/utils/utils.ts

```diff
@@ -204,7 +204,12 @@ export function login(username?: string, password?: string, firstLogin = false):
         cy.visit(Cypress.env("tackleUrl"), { timeout: 120 * SEC });
         cy.wait(5000);
         cy.get("h1", { timeout: 120 * SEC, log: false }).then(($title) => {
-            if ($title.text().toString().trim() !== "Sign in to your account") {
+            // With auth disabled, login page is not displayed and users are taken straight
+            // to the Application Inventory page.
+            if (
+                $title.text().toString().trim() !== "Sign in to your account" &&
```

Comment:
Please check this with a new cluster where password has not been changed . I think this will fail there .

Reply:
Unfortunately, I don't have a freshly deployed cluster because of cluster deployment issues .

Reply:
Ran the PR on a freshly deployed MTA set up that required admin password to be updated upon first login.  
The PR passed .